src/common/entities.ts

```diff
@@ -73,7 +73,7 @@ export interface SelectCategoryValueView {
  */
 export interface SelectCategoryView {
   isDisabled?: boolean;
-  key: CategoryValueKey;
+  key: CategoryKey;
```

Comment:
I'm pretty sure the types of the `key` properties for `SelectCategoryView` and `SelectCategoryValue` were backwards, but it didn't cause any errors because they were both equivalent; since I've changed `CategoryValueKey` to `unknown`, I've switched them around to (hopefully) be accurate

Reply:
Agree - thank you for picking this up!